docs/api-reference/core/geometries/cone-geometry.md

```diff
@@ -2,6 +2,8 @@
 
 Creates a cone geometry.
 
+The generated geometry will have `indices` and `POSITION`, `NORMAL` and `TEXCOORD_0` attributes.
```

Comment:
I think the attribute names in the geometries should follow the Luma.gl naming conventions, since they are Luma classes. Is this convention from loaders.gl? Maybe we just need a function `getGeometryFromLoadersgl`. Though I'm thinking loaders.gl should use the Luma naming conventions to keep the whole suite of tools on the same page?

Reply:
Update per discussion - we'll gradually replace the old luma.gl conventions with glTF naming.